staging/src/k8s.io/apiserver/pkg/server/resourceconfig/resource_config.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package storage
+package resourceconfig
```

Comment:
do we have to move the interfaces? It would make this PR much smaller if we didn't.

Reply:
Not a must.